replicator/src/main/scala/com/evolutiongaming/kafka/journal/replicator/DistributeJob.scala

```diff
@@ -16,6 +16,30 @@ import com.evolutiongaming.skafka.{Partition, Topic, TopicPartition}
 
 import scala.concurrent.duration.*
 
+/**
+ * [[DistributeJob]] is a service that distributes jobs across application nodes.
+ * 
+ * Distribution done with help of Apache Kafka consumers working within a single consumer group.
```

Comment:
Shall we mention that it is recommended to setup topic with higher than minimal number of partitions? 

For example, topic with 3 partitions will not distribute job across 13 nodes (10 nodes will be job-less) as topic with 39 partitions.

Shall we add docs also about `Distributed#single` here or on its definition?

Reply:
thanks for the advice, done